lib/uniffi-zcash/src/udl/secp256k1/secret_key.udl

```diff
@@ -0,0 +1,6 @@
+interface SecpSecretKey {
+    [Throws=ZcashError]
+    constructor(sequence<u8> data);
+
+    sequence<u8> serialize_secret();
+};
```

Comment:
Here we are leaving the zcash domains and started mapping the [secp256k1](https://docs.rs/secp256k1/0.27.0/secp256k1/struct.SecretKey.html) crate. This is being done because this object its being returned by some methods of the `ZcashAccountPrivKey` object.

Its not the intention to provide a full mapping of this object. Just the very basic functionality to allow the user to serialize it for later parsing with a `secp256k1` library of each language in case is necessary.

Reply:
Ok.